app/src/main/java/org/p2p/wallet/utils/LamportsAmount.kt

```diff
@@ -8,3 +8,4 @@ data class LamportsAmount(val value: BigDecimal) {
 }
 
 fun BigDecimal.toLamportsInstance(): LamportsAmount = LamportsAmount(this)
+fun Double.toLamportsInstance(): LamportsAmount = toBigDecimal().toLamportsInstance()
```

Comment:
This is incorrect class. 
Lamports mean `BigInteger` 
What is the idea of this class? @gslevinkov

Reply:
oh yeah, sorry, i made a mistake, the inside value should be BigInteger.
Jupiter return values in Lamports, but the type in API is double and I thought it would be big decimal.

the idea is use types instead of god know what like we do with Base58String and Base64String, we always know what format string has and compiler helps us. 
LamportsAmount - same idea. instead of adding `InLamports` everywhere - we can use type that tells us that this value / parameter - is in Lamports.